test/integration/restart_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'test_helper'
+require 'kubernetes-deploy/restart_task'
+
+class KubernetesDeployTest < KubernetesDeploy::IntegrationTest
+  def test_restart
```

Comment:
That's kind of hard to reproduce. I should have mentioned it as the reason why I didn't add an error case.